magenta/models/shared/melody_rnn_generate.py

```diff
@@ -59,6 +59,10 @@
                            'will be used as a priming melody. If a primer '
                            'melody is not specified, melodies will be '
                            'generated from scratch.')
+tf.app.flags.DEFINE_float('bpm', 0,
+                          'The beats per minute to play generated output at. '
+                          'If 0, bpm will default to 120. If the primer has a '
+                          'bpm that will be used instead.')
```

Comment:
it looks like this value is not used if the primer is a MIDI file? can you clarify?